### PR TITLE
Container for experiment reports

### LINKED ISSUE #24288: [UI] Container and Route for Reports of Experiment
### Parent Issue

https://github.com/dotCMS/core/issues/23625

### User Story

As a user, I want to be able to go to reports of experiments, 

### Acceptance Criteria

- Router working
- Container with the Name of the experiment
- Content with an empty body

### Proposed Objective

User Experience

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

Screenshot 2023-03-06 at 4 08 16 PM

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by oidacra**
### Proposed Changes
* Create a new route
* Create a new container component
* Create a new store
* Create a new icon in Experiments Listing to access to Experiment Reports

### Checklist
- [x] Tests
- [x] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
Just a container for use in reports issues.

### Screenshots
**New icon in Listing Experiments**
Screenshot 2023-03-10 at 10 52 47 AM

**Container**
Screenshot 2023-03-10 at 11 01 31 AM

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+            .content {
+                width: $container-reports-max;
+            }
+        }
+    }
+}
+
+::ng-deep {
+    p-tag .p-tag {
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss:37]: Can we make this global somehow? not sure how...

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
+@use "variables" as *;
+
+:host {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    padding-bottom: $spacing-7;
+
+    .dot-experiment-report-content {
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss:10]: Not needs nesting here.

**Review by fmontes — DISMISSED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.spec.ts [hunk 3]
```diff
@@ -88,6 +88,29 @@ const archivedExperiments: DotExperiment[] = [
         goals: null
     }
 ];
+
+const runningExperiments: DotExperiment[] = [
```
Comment (hmoreras) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.spec.ts:92]: why don't use the current mocks and just overwrite the property in the test ?

Comment (hmoreras) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.spec.ts:92]: why don't use the current mocks and just overwrite the property in the test ?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    });
+
+    it('should show the skeleton component when is loading', () => {
+        spectator.component.vm$ = of({ ...defaultVmMock, isLoading: true });
+        spectator.detectChanges();
+
+        expect(spectator.query(DotExperimentsUiHeaderComponent)).toExist();
+        expect(spectator.query(DotExperimentsReportsSkeletonComponent)).toExist();
+    });
+    it('should show the skeleton component when is not loading', () => {
```
Comment (hmoreras) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.spec.ts:83]: fix description

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-shell/dot-experiments-shell-routing.module.ts [hunk 5]
```diff
@@ -29,6 +29,13 @@ const routes: Routes = [
                             '../dot-experiments-configuration/dot-experiments-configuration.module'
                         )
                     ).DotExperimentsConfigurationModule
+            },
+            {
+                path: 'reports',
+                title: 'Experiment Reports',
```
Comment (hmoreras) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-shell/dot-experiments-shell-routing.module.ts:35]: the tittles in this routes need `internationalization` ?

**Review by hmoreras — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss (hunk 1 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss:37]: Moved to _tag.scss with comments to separate from the default statuses.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss (hunk 2 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss:10]: Moved.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.spec.ts (hunk 4 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.spec.ts:83]: fixed

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-shell/dot-experiments-shell-routing.module.ts (hunk 5 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-shell/dot-experiments-shell-routing.module.ts:35]: @fmontes? I did this `TitleStrategy` (`core-web/apps/dotcms-ui/src/app/shared/services/dot-title-strategy.service.ts`), where we can inject the pipe (not sure) and translate it direct like the directive of `DotDropdown`

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.spec.ts (hunk 3 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.spec.ts:92]: Updated.

**Review by hmoreras — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.ts [hunk 6]
```diff
@@ -66,4 +69,14 @@ export class DotExperimentsListTableComponent {
...
     }
+
+    /**
+     * Go to report of experiment
+     * @param {DotExperiment} item
+     * @returns void
+     * @memberof DotExperimentsListTableComponent
+     */
+    viewReports(item: DotExperiment) {
+        this.reportItem.emit(item);
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.ts:80]: test of this `emit`?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/dot-experiments-list.component.ts [hunk 7]
```diff
@@ -118,4 +118,21 @@ export class DotExperimentsListComponent {
...
     }
+
+    /**
+     * Go to view Experiment Report
+     * @param {DotExperiment} experiment
+     * @returns void
+     * @memberof DotExperimentsShellComponent
+     */
+    goToViewExperimentReport(experiment: DotExperiment) {
+        this.router.navigate(['/edit-page/experiments/reports/', experiment.id], {
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/dot-experiments-list.component.ts:129]: test?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-reports-skeleton/dot-experiments-reports-skeleton.component.html [hunk 8]
```diff
@@ -0,0 +1,46 @@
+<!--  summary  -->
+<div class="grid" style="margin-top: 1.5rem; margin-bottom: 1.5rem">
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-reports-skeleton/dot-experiments-reports-skeleton.component.html:2]: move `grid` styles to scss file

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-reports-skeleton/dot-experiments-reports-skeleton.component.html [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+            </div>
+            <div class="col-3 col-offset-1">
+                <p-skeleton height="1rem" width="100%"></p-skeleton>
+            </div>
+        </div>
+    </div>
+</div>
+
+<!--  chart  -->
+<div class="grid" style="margin-top: 1rem">
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-reports-skeleton/dot-experiments-reports-skeleton.component.html:22]: https://github.com/dotCMS/core/pull/24345/files#r1142701737

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.spec.ts [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+        expect(spectator.query(DotExperimentsUiHeaderComponent)).toExist();
+        expect(spectator.query(DotExperimentsReportsSkeletonComponent)).toExist();
+    });
+
+    it("shouldn't show the skeleton component when is not loading", () => {
+        spectator.component.vm$ = of({ ...defaultVmMock, isLoading: false });
+        spectator.detectChanges();
+
+        expect(spectator.query(DotExperimentsUiHeaderComponent)).toExist();
+        expect(spectator.query(DotExperimentsReportsSkeletonComponent)).not.toExist();
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.spec.ts:89]: missing this test `dot-experiments-experiment-summary`

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.ts [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    /**
+     * Go to Experiment List
+     * @param {string} pageId
+     * @returns void
+     * @memberof DotExperimentsReportsComponent
+     */
+    goToExperimentList(pageId: string) {
+        this.router.navigate(['/edit-page/experiments/', pageId], {
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.ts:60]: test?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/store/dot-experiments-reports-store.spec.ts [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+            experiment: EXPERIMENT_MOCK,
+            status: ComponentStatus.IDLE
+        };
+
+        expect(dotExperimentsService.getById).toHaveBeenCalledWith(EXPERIMENT_MOCK.id);
+        store.state$.subscribe((state) => {
+            expect(state).toEqual(expectedInitialState);
+            done();
+        });
+    });
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/store/dot-experiments-reports-store.spec.ts:66]: missing tests of updaters, effects, etc

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-reports-skeleton/dot-experiments-reports-skeleton.component.html (hunk 9 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-reports-skeleton/dot-experiments-reports-skeleton.component.html:22]: done.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.ts (hunk 6 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.ts:80]: added

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/dot-experiments-list.component.ts (hunk 7 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/dot-experiments-list.component.ts:129]: Added.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.spec.ts (hunk 10 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.spec.ts:89]: Added.

**Review by alfredo-dotcms — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-shell/dot-experiments-shell-routing.module.ts (hunk 5 above)
Comment (oidacra) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-shell/dot-experiments-shell-routing.module.ts]: translated.

**Pull request merged**